compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1364,13 +1365,20 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
 
         let import_suggestions =
             self.lookup_import_candidates(ident, Namespace::MacroNS, parent_scope, is_expected);
+        let (span, found_use) =
+            if let Some(def_id) = parent_scope.module.nearest_parent_mod().as_local() {
+                UsePlacementFinder::check(krate, self.def_id_to_node_id[def_id])
+            } else {
+                (None, FoundUse::No)
+            };
```

Comment:
Nit: `if let` with two short alternatives typically looks better and more compact as a `match`.